spring/README.rst

```diff
@@ -0,0 +1,101 @@
+***************
+Notejam: Spring
+***************
+
+Notejam application implemented using `Spring <http://projects.spring.io/spring-framework/>`_ framework.
+
+Spring version: 4.2.3
+
+The full stack would be:
+
+- `Spring Boot <http://projects.spring.io/spring-boot/>`_ (Spring configuration)
+- `Thymeleaf <http://www.thymeleaf.org/>`_ (View)
+- `Spring Security <http://projects.spring.io/spring-security/>`_ (Security framework)
+- `Spring`_ (DI and MVC framework)
+- `Spring Data <http://projects.spring.io/spring-data/>`_ (Persistence abstraction)
+- `JPA <http://www.oracle.com/technetwork/java/javaee/tech/persistence-jsp-140049.html>`_ (Persistence API)
+- `Hibernate <http://hibernate.org/orm/>`_ (JPA implementation)
+
+==========================
+Installation and launching
+==========================
+
+-----
+Clone
+-----
+
+Clone the repo:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/komarserjio/notejam YOUR_PROJECT_DIR/
+
+-------
+Install
+-------
+
+Install a `JDK <http://openjdk.java.net/>`_ and `Maven <https://maven.apache.org/>`_.
+
+-------------
+Configuration
+-------------
+
+The application has a password recovery process which involves sending an email.
+If you want to enable that, you have to create a local application.properties file
+and set there the property spring.mail.host to your SMTP server (e.g. spring.mail.host = smtp.example.net).
```

Comment:
Per default there's no mail transport. I didn't find any implementation for a file transport. Do you need such an implementation?

For testing you could also simply add the MX of the receiving email address (e.g. gmail-smtp-in.l.google.com) into your local application.properties file.